recipe/meta.yaml

```diff
@@ -43,41 +36,41 @@ outputs:
         - python
         - pip
         - certifi >=2020.06.20
+        - freetype >=2.3
+        - numpy
+        - setuptools
         - setuptools_scm >=4
         - setuptools_scm_git_archive
-        - numpy
-        - cycler >=0.10
-        - python-dateutil >=2.7
-        - freetype
-        - nose
-        - pyparsing >=2.2.1
         - tk  # [linux]
       run:
-        - python
-        - setuptools
-        - {{ pin_compatible('numpy') }}
-        - certifi >=2020.06.20
+        - python >=3.7
```

Comment:
This should be controlled by the exports of the `python` in the host section and build skips. What is causing the need for a pin here?

Reply:
It's my mistake. I've just fixed it